src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return total - 20;
+  }
+
+  if (days >= 7) {
+    return total - 50;
+  }
```

Comment:
PAy attention to comments, [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T18%3A34%3A59Z) 1st item. You should create variables for main values - 20, 50, 40, 3, 7 they shouldn't be just magic numbers